optimism.tokenlist.json

```diff
@@ -707,6 +707,28 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 42,
+      "address": "0x002be8a5961e0f352092d6693133a6944b7846ba",
+      "name": "Perpetual",
+      "symbol": "PERP",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/PERP.png",
```

Comment:
Can you pls add a smaller image of maximum size 256x256. We can test with this one as well but I suspect it will not squash properly.

Reply:
already change size to 256X256